lizmap/modules/presentation/install/upgrade.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @author    3liz
+ * @copyright 2011-2019 3liz
+ *
+ * @see      http://3liz.com
+ *
+ * @license   Mozilla Public License : http://www.mozilla.org/MPL/
+ */
+class presentationModuleUpgrader extends jInstallerModule
```

Comment:
you should use the new classes of Jelix 1.8  `\Jelix\Installer\Module\Installer`